engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbContext.java

```diff
@@ -45,32 +44,39 @@ public void initialize(String preInitializeLanguages) {
     if (!isInner) {
       if (innerContext == null) {
         innerContext =
-            new GuardedTruffleContext(
-                env.newInnerContextBuilder()
-                    .initializeCreatorContext(true)
-                    .inheritAllAccess(true)
-                    .config(INNER_OPTION, "yes")
-                    .build(),
-                true);
+            env.newInnerContextBuilder()
+                .initializeCreatorContext(true)
+                .inheritAllAccess(true)
+                .config(INNER_OPTION, "yes")
+                .build();
       }
-      initializeLanguages(env, innerContext, preInitializeLanguages);
+      // initializeLanguages(env, innerContext, preInitializeLanguages);
```

Comment:
Leftover? Remove `initializeLanguages`?

Reply:
Removed in ef1c79825a48414f0e9678a4917faf4bcc6d53be